.github/ISSUE_TEMPLATE/ad-hoc-meeting.md

```diff
@@ -3,7 +3,6 @@ name: ad-hoc-meeting
 about: Used to organize ad-hoc meetings
 title: "[Ad hoc meeting] "
 labels: ''
-assignees: 'brianwarner'
```

Comment:
instead, can we make the CPC team be the default assignee?

Reply:
Brian was assigned for operational reasons (adding it to the agenda and maybe setting up zoom?). So no need to ping the CPC instead.

Reply:
Ok, then who’s doing the operational steps now?

Reply:
@ljharb you cannot assign teams on GitHub Issue templates.

Reply:
We can add the agenda label if needed.

Reply:
perhaps @bensternthal should be the assignee?

Reply:
+1 on @bensternthal he is the logical replacement

Reply:
Happy to help feel free to put my name in there.